pkg/segment/structs/evaluationstructs.go

```diff
@@ -2364,6 +2442,10 @@ func (self *TextExpr) GetFields() []string {
 		if self.LengthExpr != nil {
 			fields = append(fields, self.LengthExpr.GetFields()...)
 		}
+		if self.MultiValueExpr != nil {
+			fields = append(fields, self.MultiValueExpr.GetFields()...)
+		}
```

Comment:
This looks good. I think you also need this logic inside `ValueExpr.GetFields()`, right? Or maybe I missed that

Reply:
ValueExpr.GetFields() already has this [logic](https://github.com/siglens/siglens/blob/5b4465a6e8298316723cf9c3543bbafa03446fe2/pkg/segment/structs/evaluationstructs.go#L1109) to call the GetFields of MultiValueExpr which will handle this.

Although, I missed this [check](https://github.com/siglens/siglens/blob/5b4465a6e8298316723cf9c3543bbafa03446fe2/pkg/segment/structs/evaluationstructs.go#L1124) and have added it.

Addressed in the latest commit.